src/tribler/core/components/torrent_checker/torrent_checker/torrentchecker_session.py

```diff
@@ -31,7 +31,7 @@
 TRACKER_ACTION_ANNOUNCE = 1
 TRACKER_ACTION_SCRAPE = 2
 
-MAX_INT32 = 2 ** 16 - 1
+MAX_INT32 = 2 ** 31 - 1
```

Comment:
NIT: The value `2147483647` is more immediately recognizable than `2 ** 31 - 1`, given its prevalence in documentation. Additionally, `2 ** 31 - 1` demands a bit more cognitive effort to understand or might require familiarity with the notation (which is not necessary as the name of the constant already states that it is related to the 32-bit version).

Reply:
As an aside, since this constant is related to C(am I correct on this?), the name `INT32_MAX` might be more appropriate.

Reply:
> The value 2147483647 is more immediately recognizable than 2 ** 31 - 1, given its prevalence in documentation.

It looks like your very personal preferences. When looking at `2 ** 31 - 1`, it is immediately clear how the value is constructed, and it is possible to understand that it is correct, intended for signed numbers and how many bits are in it, while for 2147483647, it is necessary to remember is it for INT or UNSIGNED INT, for example.

> As an aside, since this constant is related to C (am I correct on this?), the name INT32_MAX might be more appropriate.

It is not specifically related to C, just to signed binary arithmetic in general. I googled for it, and while both "MAX_INT32" and "INT32_MAX" are used, "INT32_MAX" returns more results, so I'll change it to "INT32_MAX".

Reply:
Wow 64k hard limit on certain IDs, great catch! {btw in the future let's not discuss variable name cosmetics too much; correctness is obviously critical}

Reply:
@kozlovsky, indeed, it's a matter of preference as already mentioned.
@synctext, please note that the comment is labeled as **NIT** and the PR has received approval, so this discussion shouldn't block progress.